py-tests/pyspark_tests.py

```diff
@@ -0,0 +1,642 @@
+from __future__ import print_function
+import pytest
+import sys
+from operator import add
+import findspark
+findspark.init()
+from pyspark import SparkContext, SparkConf, SQLContext, Row
+import docker, os, subprocess, json, riak, time
+import pyspark_riak
+import timeout_decorator
+import datetime
+#sudo route add 172.17.0.0/16 192.168.99.100, osx only
+#docker, pull riak-ts image
+#pytest
+#cd to the test file and run, py.test pyspark_tests.py -s
+#pass in uber jar as os.environ['SPARK_CLASSPATH']
+
+os.environ['SPARK_CLASSPATH'] = '/Users/basho/.m2/repository/com/basho/riak/spark-riak-connector/1.5.1-SNAPSHOT/spark-riak-connector-1.5.1-SNAPSHOT-uber.jar'
```

Comment:
Generally, do not set environment variables that are already set, so add 
```python
if os.environ['SPARK_CLASSPATH']:
```

above the default setting. The reason should be super apparent as the path is being set to a specific user's home directory which means setup steps would absolutely be needed to create the user, setup maven for the user, ....